apps/app/src/hooks/use-rss-feed.ts

```
@@ -0,0 +1,137 @@
+
+  return { title, description, link, items };
+}
+
+export function useRssFeed(feedId: string) {
+  const { data: feedData } = useFeed(feedId);
+
+  const rssFeed =
+    feedData &&
+    feedData.config.outputs.stream?.distribute?.find(
+      (distribute) => distribute.plugin === "@curatedotfun/rss",
+    );
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)